test/DynamoCoreWpfTests/PreferencesGeometryScalingTests.cs

```diff
@@ -0,0 +1,91 @@
+
+            //Change the RunType to Automatic, so when the MarkNodesAsModifiedAndRequestRun() is called a graph execution will be kicked off 
+            View.RunSettingsControl.RunTypesComboBox.SelectedItem = View.RunSettingsControl.RunTypesComboBox.Items[1];
+            DispatcherUtil.DoEvents();
+
+            //Creates the Preferences dialog and the ScaleFactor = 2 ( Medium)
+            var preferencesWindow = new PreferencesView(View);
+            preferencesWindow.Show();
+            DispatcherUtil.DoEvents();
+
+            //Change the RadioButton checked so the ScaleFactor is updated to -2 (Small)
+            preferencesWindow.RadioSmall.IsChecked = true;
+            DispatcherUtil.DoEvents();
+
+            //Close the Preferences Dialog and due that the ScaleFactor was updated the MarkNodesAsModifiedAndRequestRun() method will be called
+            preferencesWindow.CloseButton.RaiseEvent(new RoutedEventArgs(ButtonBase.ClickEvent));
+            DispatcherUtil.DoEvents();
+
+            //This will get the Circle.ByCenterPointRadius node (connected to the CodeBlock)
+            var nodeView = NodeViewWithGuid("bcf1c893-5311-4dff-9a04-203ffb9d426c");
+
+            //When RunType = Automatic when the graph is executed the ByCenterPointRadius node change status to Warning due that ScaleFactor = Small
+            Assert.AreEqual(nodeView.ViewModel.State, ElementState.Warning);
+        }
+
+        /// <summary>
+        /// The test validates that when RunType=Manual and the MarkNodesAsModifiedAndRequestRun() method is called the graph execution is NOT kicked off
+        /// </summary>
+        [Test]
+        public void PreferencesGeoScaling_RunGraph_Manual_Mode()
+        {
+            //The GeometryScalingCodeBlock.dyn contains a CodeBlock with a large number that needs ScaleFactor > Medium
+            Open(@"core\GeometryScalingCodeBlock.dyn");
+
+            //Creates the Preferences dialog and the ScaleFactor = 2 ( Medium)
+            var preferencesWindow = new PreferencesView(View);
+            preferencesWindow.Show();
+            DispatcherUtil.DoEvents();
+
+            //Change the RadioButton checked so the ScaleFactor is updated to -2 (Small)
+            preferencesWindow.RadioSmall.IsChecked = true;
+            DispatcherUtil.DoEvents();
+
+            //Close the Preferences Dialog and due that the ScaleFactor was updated the MarkNodesAsModifiedAndRequestRun() method will be called
+            preferencesWindow.CloseButton.RaiseEvent(new RoutedEventArgs(ButtonBase.ClickEvent));
+            DispatcherUtil.DoEvents();
+
+            //This will get the Circle.ByCenterPointRadius node (connected to the CodeBlock)
+            var nodeView = NodeViewWithGuid("bcf1c893-5311-4dff-9a04-203ffb9d426c");
+
+            //When RunType = Automatic when the graph is executed the ByCenterPointRadius node change status to Warning due that ScaleFactor = Small
```

Comment:
This comment doesn't apply here. Needs to be updated for this case.

Reply:
@aparajit-pratap the comment was updated in commit: https://github.com/DynamoDS/Dynamo/pull/11867/commits/688b825e39d3edfe446131556757a59360040a5b